### PR TITLE
Reverse singly LinkedList problem solution was added

### PR DESCRIPTION
**Pull request by Ildar10**

### CODE REVIEW

File: Practice Problems/LinkedList/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# LinkedList
+All questions added here are taken from various sites like GeeksForGeeks, LeetCode, Hackerrank, CodeForces, etc.
+
+## Problems
+- [Hamming Distance](https://github.com/srsandy/Data-Structures-and-Algorithms-in-Java-2nd-Edition-by-Robert-Lafore/tree/master/Practice%20Problems/LinkedList/Reverse%20LinkedList)
```
Comment (srsandy) [inline Practice Problems/LinkedList/README.md]: Can you pls rename this to Reverse LinkedList and not Hamming Distance

Comment (Ildar10) [reply Practice Problems/LinkedList/README.md]: > Can you pls rename this to Reverse LinkedList and not Hamming Distance

done

**Comment by srsandy** [comment]
@Ildar10  thanks for the PR.
Appreciate it.

**Comment by Ildar10** [comment]
> @Ildar10 thanks for the PR.
> Appreciate it.

welcome :) could you please merge this PR?

**Pull request merged**